src/Field/ScaleCombo/ScaleCombo.jsx

```diff
@@ -164,32 +249,11 @@ class ScaleCombo extends React.Component {
    *
    * @return {Element} Option element for provided zoom level
    */
-  determineOptionKeyForZoomLevel (zoom) {
-    if (!isNumber(zoom) || (this.state.scales.length - 1 - zoom) < 0) {
+  determineOptionKeyForZoomLevel = (zoom) => {
+    if (!isInteger(zoom) || (this.state.scales.length - 1 - zoom) < 0) {
       return undefined;
     }
-    return this.state.scales[this.state.scales.length - 1 - zoom].key;
-  }
-
-  /**
-   * handleOnKeyDown of input:
-   * trigger setScale (passed by props) after ENTER key pressed
-   *
-   * @param {Event} event KeyBoard event
-   */
-  handleOnKeyDown = (event) => {
```

Comment:
I'm not sure whether this was needed for IE11?! Can you check if selection works in IE11 as well if Enter-key is pressed, please?

Reply:
Currently none of our examples is working in IE11 so it's hard to debug. I'll merge this and open an Issue regarding IE.